docs/src/rsh/errors/index.md

```diff
@@ -2163,7 +2163,7 @@ For example, the code below erroneously uses the same name multiple times:
 
 ```reach
 const A = Participant('Flower_girl', {});
-const B = API('Flower', { girl: Fun([UInt], Null) });
+const B = API('Flower_girl', { girl: Fun([UInt], Null) });
```

Comment:
This is not a correct change. The API scope is part of the names that get generated. Your change would make the function called `Flower_girl_girl`